pkgs/development/compilers/rust/rustc.nix

```diff
@@ -21,7 +21,7 @@ let
   inherit (lib) optionals optional optionalString concatStringsSep;
   inherit (darwin.apple_sdk.frameworks) Security;
 in stdenv.mkDerivation rec {
-  pname = "rustc";
+  pname = "${pkgsBuildTarget.targetPackages.stdenv.cc.targetPrefix}rustc";
```

Comment:
a bit unrelated to this change: maybe we could extract ``pkgsBuildTarget.targetPackages.stdenv.cc`` into a variable to short this a bit